tests/hpc/ganglia_client.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Ganglia Test - client
+#   Acts as client node, which publishes data to the server via gmetric command
+# Maintainer: soulofdestiny <mgriessmeier@suse.com>
+# Tags: https://fate.suse.com/323979
+
+use base "hpcbase";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    my ($server_ip) = get_required_var('HPC_MASTER_IP') =~ /(.*)\/.*/;
+
+    # set proper hostname
+    assert_script_run('hostnamectl set-hostname ganglia-client');
+
+    # Prepare Client by installing "ganglia-gmond"
+    zypper_call 'in ganglia-gmond';
+
+    # Start gmond on Client
```

Comment:
I think with these comments you are *a bit* too explicit/redundant. I would delete the blank lines and the comments in these three commend